docs/use_cases/model_applications/s2s_mid_lat/UserScript_fcstGFS_obsERA_Blocking.py

```diff
@@ -1,178 +1,251 @@
 """
-Blocking Calculation: GFS and ERA RegridDataPlane, PcpCombine, and Blocking python code
-========================================================================================
+UserScript and StatAnalysis: Calculate and evaluate Blocking for the GFS and ERA
+================================================================================
 
 model_applications/
```

Comment:
```suggestion
model_applications/s2s_mid_lat/UserScript_fcstGFS_obsERA_Blocking.py
```
The [template](https://github.com/dtcenter/METplus/blob/develop/docs/use_cases/use_case_documentation_template.py) has the full path on one line.